lib/constants.js

```diff
@@ -108,6 +108,9 @@ const PEGIN_EVENTS = {
 
 const PEGIN_V1_RSKT_PREFIX_HEX =  '52534b54';
 
+const MINIMUM_PEGOUT_AMOUNT_IN_SATOSHIS = 250_000;
```

Comment:
Nice idea. But way out of the scope. We could create an analysis ticket to see how we would handle that.

Reply:
That's a nice idea, we could make use of the btcEthUnitConverter library to build that class and do some conversions. It could even be its own library perhaps